test/modules/TestResultCoordinator/TestUtil.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json.Linq;
+    using TestResultCoordinator.Reports;
+    using TestResultCoordinator.Reports.DirectMethod;
+
+    static class TestUtil
```

Comment:
Suggestion: rename to Util or ReportGeneratorUtil

Reply:
It is not only used for report generation; how about TestReportUtil.